sharing/present_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2021 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package sharing
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+
+	"storj.io/linksharing/objectmap"
+	"storj.io/uplink"
+)
+
+func TestDownloadContentTypeHeader(t *testing.T) {
+	cfg := Config{
+		URLBase:   "http://test.test",
+		Templates: "../web",
+	}
+
+	handler, err := NewHandler(&zap.Logger{}, &objectmap.IPDB{}, cfg)
+	require.NoError(t, err)
+
+	ctx := context.Background()
```

Comment:
Usually I prefer using testcontext, because it protects against test stalls and long tests.